### PR TITLE
Add a known issue in OCP 4.16 Release Notes

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.16

Issue:

Link to docs preview:
https://78149--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-known-issues_release-notes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3178,6 +3178,8 @@ In the following tables, features are marked with the following statuses:
 
 * Enabling LUKS encryption on a system using 512 emulation disks causes provisioning to fail and the system launches the emergency shell in the initramfs. This happens because of an alignment bug in `sfdisk` when growing a partition. As a workaround, you can use Ignition to perform the resizing instead. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
 
+* {product-title} {product-version} disconnected installation fails on {ibm-power-name} Virtual Server.
```
Comment (maxwelldb) [inline release_notes/ocp-4-16-release-notes.adoc:3181]:
```suggestion
* {product-title} version {product-version} disconnected installation fails on {ibm-power-name} Virtual Server.
```

Comment (maxwelldb) [reply release_notes/ocp-4-16-release-notes.adoc:3181]: Is there an issue or a workaround to add to this?

**Review by maxwelldb — COMMENTED** [review]
Left a suggestion and a question.

Note: This documents a private KI. Should not be merged unless the ticket's sec level is reclassed.

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc:3181]: the build fails everytime when I add the issue link as mentioned below. 

```
* {product-title} {product-version} disconnected installation fails on {ibm-power-name} Virtual Server. 
(link:https://issues.redhat.com/browse/OCPBUGS-36250[*OCPBUGS-36250*])
```

Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc]: fyi : @maxwelldb could you please suggest what can be done in this case.

Comment (alishaIBM) [reply release_notes/ocp-4-16-release-notes.adoc]: >Is there an issue or a workaround to add to this?
@maxwelldb I have now added the issue link.

**Pull request merged**